docs/OnnxReleases.md

```diff
@@ -115,6 +115,9 @@ The ONNX project, going forward, will plan to release roughly on a four month ca
 **Merge into main branch**
 * After everything above is done, merge the release branch into the main branch to make it consistent. This step is needed only when there are urgent changes that are made directly into the release branch. The main branch does not have these needed changes. In all other circumstances, the merge PR shall show as empty so nothing needs to be merged.
 
+**Update PyPI account credentials**
+* To secure, [update](https://pypi.org/manage/account/) old PyPI account credentials after every release. Let steering committee know the updated credentials for future use.
```

Comment:
"Let steering committee know the updated credentials for future use." -> "Update the credential repository and let the Steering Committee know once you are done"